packages/os/os.spec

```diff
@@ -31,6 +31,7 @@ Source12: 00-resolved.conf
 %if %{with k8s_runtime}
 Source13: cis-checks-k8s-metadata-json
 %endif
+Source18: bootstrap-containers-toml
```

Comment:
The weird numbering here is because of additional configuration file pull requests in flight?

Reply:
Correct